### ISSUE TITLE
Incorrect path to PowerShell.exe when mounting point for drive C isn't default in WSL 2

### ISSUE THREAD

**Issue by kalexey89** [sindresorhus/open#218]
The #217 PR broke ability to start ReactJS app in dev mode for me.  When trying to start it using `yarn start` command I get an error:
```
events.js:292
      throw er; // Unhandled 'error' event
      ^

Error: spawn /mnt/c/Windows/System32/WindowsPowerShell/v1.0/powershell.exe ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:269:19)
    at onErrorNT (internal/child_process.js:465:16)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)

...
```

As we can see from message - `powershell.exe` not found. That happened because I'm didn't use default mounting point for drive C. My mounting point for C is different - `/c` instead of `/mnt/c`.
But the #217 PR change dynamically determination of mounting point to constant ([see 53 line](https://github.com/sindresorhus/open/blob/5ce319c8482a2c6425e0514e0da1630dae85a986/index.js#L53))
Mounting point should be determined dynamically as it be in v7.3.0.

**Comment by sindresorhus** [comment]
// @anaisbetts

**Comment by anaisbetts** [comment]
I'll see what I can do tomorrow

**Comment by kalexey89** [comment]
@anaisbetts The #219 PR will fix this problem. Please review it. Thanks in advance!

**Comment by kalexey89** [comment]
Fixed by #219 PR.
